packages/salesforcedx-apex-replay-debugger/src/breakpoints/index.ts

```diff
@@ -13,3 +13,8 @@ export interface LineBreakpointInfo {
 export { BreakpointUtil } from './breakpointUtil';
 import { BreakpointUtil } from './breakpointUtil';
 export const breakpointUtil = BreakpointUtil.getInstance();
+
+export interface LineBreakpointEventArgs {
```

Comment:
The line breakpoint event response now contains the line breakpoint information and the project path. This is the interface.